plugins/last-fm/back.js

```diff
@@ -0,0 +1,192 @@
+const fetch = require('node-fetch');
+const md5 = require('md5');
+const open = require("open");
+const axios = require('axios');
+const { setOptions } = require('../../config/plugins');
+const getSongInfo = require('../../providers/song-info');
+
+const defaultSettings = {
+	enabled: true,
+	api_root: "http://ws.audioscrobbler.com/2.0/",
+	api_key: "04d76faaac8726e60988e14c105d421a", // api key registered by @semvis123
+	secret: "a5d2a36fdf64819290f6982481eaffa2",
+	suffixesToRemove: [' - Topic', 'VEVO']
+}
+
+const cleanupArtistName = (config, artist) => {
+	// removes the suffixes of the artist name for more recognition by last.fm
+	let { suffixesToRemove } = config;
+	if (suffixesToRemove === undefined) {
```

Comment:
nit: it does not seem necessary to override the suffixes if not defined, we can just return early:
```suggestion
	if (suffixesToRemove === undefined) {
	    return artist;
	}
```